db_startup.py

```diff
@@ -0,0 +1,56 @@
+from db_classes import TaskType, Field, ValueType, User
+from db_helper import connectToDatabase, disconnectFromDatabase
+
+
+
+# NOTE: currently not in use
+# # function that creates the admin user and saves it to the database 
+# def createAdminUser():
+#     connection = connectToDatabase()
+
+#     admin_user = User(username="admin", display_name="admin", hash_password="password", hash_salt="password", workspaces=[], tasks=[], owned_filtered_views=[], shared_filtered_views=[])
+#     admin_user.save()
+
+#     disconnectFromDatabase(connection)
+
+
+# function that creates the default task type and saves it to the database
+def createDefaultTaskType():
+
+    # create a Tag Value Type
+    tag_value_type = ValueType(name="Tag", allowed_values=[])
+    tag_value_type.save()
+
+    # create default ValueTypes: string and user
+    string_value_type = ValueType(name="String", allowed_values=[])
+    user_value_type = ValueType(name="User", allowed_values=[])
```

Comment:
Something for a future discussion/investigation, but I wonder if we could get away with nulling out allowed_values for these sorts of value types which don't have a predefined list of values.